phoenicis-scripts/src/main/java/org/phoenicis/scripts/engine/injectors/IncludeInjector.java

```diff
@@ -22,16 +22,18 @@ public IncludeInjector(ScriptFetcher scriptFetcher) {
     public void injectInto(PhoenicisScriptEngine phoenicisScriptEngine) {
         final Set<String> includedScripts = new HashSet<>();
 
-        phoenicisScriptEngine.put("include", (Consumer<String>) argument -> {
+        phoenicisScriptEngine.put("include", (Function<String, Object>) argument -> {
```

Comment:
Will this work in cases where the script e.g. also defines variables? Does it affect somehow the way `include` is used in scripts?

Reply:
> Will this work in cases where the script e.g. also defines variables? Does it affect somehow the way include is used in scripts?

What do you mean by variables? This change only changes the semantics of the `include(<scriptId>)` command. To be more exact it changes the signature of the method from:

```java
void include(String scriptId);
```

to

```java
Object include(String scriptId);
```

All our existing scripts shouldn't be affected by this because they don't use the "result" of the `include` command.

Reply:
I was thinking of something like:
```Javascript
var a = "hello world";

new OnlineInstallerScript() {
    ...
}
```
Is `a` still accessible if I include this?

Reply:
It should be. I mean the script is still fully executed. The only change is, that the result of the last "line" is returned as the result of the `include(script)` command.

Reply:
Does this mean that the following would not work?
```Javascript
class MyVerb { ... }
var a = "test";
```

Reply:
Yes

Reply:
Should we use an explicit `return`? Like `return new OnlineInstallerScript`? I think this would be clearer then deliberately usin the last statement. Also it's in line with similar stuff in other script languages e.g. Lua modules.

Reply:
`return` doesn't work, because it is not a function from which we are returning.
The equivalent would be an `export`, but when using an `export` you need to use a named `import` when using the exported expression. We can't use `import` because it requires a name of the imported expression, which we don't have/know.

Reply:
I see. It probably would make sense to split the change for `include` and the installer simplification as we cannot merge the installer simplification before all scripts are done. It would also allow us to test all types (plugins, verbs etc.) first.

Reply:
See #2004